brave/browser-laptop
# Fix mysterious searx favicon disappearing in builds

fix https://github.com/brave/browser-laptop/issues/14653

for some reason, renaming the searx favicon to something that doesn't have
'searx' in the filename makes it reappear. note this is only a bug in packaged builds.

Test Plan:
1. make a packaged build
2. open brave
3. hit ':x' in urlbar. searx favicon should appear.

## Submitter Checklist:


## Test Plan:


## Reviewer Checklist:


Tests
